client/src/store/slices/login/index.ts

```diff
@@ -45,8 +46,12 @@ export const googleLogin = createAsyncThunk(
       const googleToken = await api.loginGoogle(accessToken);
       return await api.login(googleToken.accessToken);
     } catch (err) {
-      console.error(err.response.data);
-      return rejectWithValue(err.response.data);
+      if (axios.isAxiosError(err)) {
+        console.error(err?.response?.data);
+        return rejectWithValue(err?.response?.data);
+      } else {
+        console.log("Unexpected error", err);
```

Comment:
you need to also rejectWithValue in the else statement, I know you're already handling this in your other branch so might want to wait for the other branch to merge with main first and then merge, or copy your changes to this branch

Reply:
yes, it would be good to finish most of it and then merge cause this PR is getting bigger